docs/odh/grafana/add_grafana_dashboard.md

```diff
@@ -0,0 +1,32 @@
+# Add a grafana dashboard to OperateFirst Grafana
+
+To be able to include your dashboards in Grafana, you will need to add a `GrafanaDashboard` custom resource. This custom resource is defined as a yaml file that downloads the dashboard contents from a specified URL and falls back to embedded json if the URL can not be resolved.
+
+An example `grafana-dashboard-example.yaml` template is shown below:
+
+```yaml
+---
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: grafana-dashboard-from-url
+  labels:
+    app: grafana
+spec:
+  name: grafana-dashboard-from-url.json
+  url: https://raw.githubusercontent.com/integr8ly/grafana-operator/master/deploy/examples/remote/grafana-dashboard.json
+  json:
+```
+
+In this template, all you need to do is update the dashboard name, the dashboard JSON file name and the URL pointing to the location of your dashboard JSON file.
```

Comment:
I would be a bit verbose here, and  also list the actual yaml paths, e.g. instead of just "dashboard name" also say "spec.name".

Reply:
yes I was actually thinking the same too. modified it